apps/nrm/fields/index.js

```diff
@@ -271,6 +271,7 @@ module.exports = {
     mixin: 'textarea',
     'ignore-defaults': true,
     formatter: ['trim', 'hyphens'],
+    labelClassName: 'visuallyhidden',
```

Comment:
```suggestion
    isPageHeading: 'true',
```
Not sure if that works because you are adjusting the actual html on the page but might be worth giving it a try

Reply:
Setting it this way results in the field's label being set as a page heading, but it still shows directly above the text field. Is there a way to put the new content in between the heading/label and the field itself to match the designs?

<img width="686" alt="NRM-387-page-heading" src="https://github.com/user-attachments/assets/b5bd074a-2eb0-4226-98c7-44a56543200b" />

Reply:
According to govuk design system it should be part of the heading https://design-system.service.gov.uk/components/textarea/

I think it works if you remove the header in translations and just leave the label.  But you'll need to change the name of the label to "Tell us...."
It's been a long time since I last developed this in hof so I can't remember exactly

The text detail is another matter

Reply:
I think the issue is here https://github.com/UKHomeOffice/modern-slavery/blob/master/apps/nrm/views/details-last-contact.html

It's a custom page

Reply:
See if that sorted it first and if it doesn't then look at the textarea